assets/question_sets/strings.js

```diff
@@ -26,5 +26,5 @@ globalData.questionSets['strings'] = {  // eslint-disable-line dot-notation
       "I will provide you with feedback."
     ].join('')
   ],
-  questionIds: ['reverseWords', 'parens', 'i18n', 'rle']
+  questionIds: ['triangle', 'reverseWords', 'parens', 'i18n', 'rle']
```

Comment:
No need to add this to strings, especially since it's not even the problem you're trying to submit. This will just cause a crash.